examples/repository/README.md

```diff
@@ -9,6 +9,9 @@ This TUF Repository Application Example has the following features:
 - Serves metadata and targets on localhost (default port 8001)
 - Simulates a live repository by automatically adding a new target
   file every 10 seconds.
+- Exposes a small API for the uploader tool example. API POST endpoints are:
+    /api/role/<ROLE>
+    /api/delegation/<ROLE>
```

Comment:
Minor nit: I don't think GH Markdown will render the newlines.

Reply:
Also, I think a bit more API documentation somewhere would be helpful (e.g. 92e03d2d2080934f1472ab0d91e533a400715117). Maybe that's too much for this doc. Docstrings in `_simplerepo.py` or `repo` would also be fine.

Reply:
I think I'll leave the API vaguely documented -- there's less maintenance and for anyone interested it's not hard to lookup the API end point implementation.

Reply:
You're right. The demo tool will likely be the only user of the API.